test/test_duo_api.rb

```diff
@@ -140,3 +142,53 @@ def test_hmac_sha1
   end
 
 end
+
+class MockResponse < Object
+    def initialize(code)
+        @code = code
+    end
+
+    def code()
```

Comment:
You can get rid of this `def code()` getter method and add `attr_reader :code` above the initialize method. It essentially does the same thing: expose the instance method on the object

Reply:
Thanks for pointing that out!